wfe/wfe.go

```diff
@@ -983,8 +984,11 @@ func (wfe *WebFrontEndImpl) Challenge(
 
 	authz, err := wfe.SA.GetAuthorization(ctx, authorizationID)
 	if err != nil {
-		// TODO(#1198): handle db errors etc
-		notFound()
+		if err == sql.ErrNoRows {
+			notFound()
+		} else {
+			wfe.sendError(response, logEvent, probs.ServerInternal(""), err)
```

Comment:
Based on @jsha's comment can you update this line to `probs.ServerInternal("Problem getting authorization")` and  the fix the corresponding test accordingly?

Thanks! I think that's the last nit before we can merge :-)